docs/subnets/hello-world-precompile-tutorial.md

```diff
@@ -548,201 +850,148 @@ func setGreeting(accessibleState PrecompileAccessibleState, caller common.Addres
+// with the registry.
+import (
+	_ "github.com/ava-labs/subnet-evm/precompile/contracts/deployerallowlist"
 
-<!-- markdownlint-enable MD013 -->
+	_ "github.com/ava-labs/subnet-evm/precompile/contracts/nativeminter"
 
-Finally, we can add our precompile in `GetActivePrecompiles()`, which given a `blockTimestamp`
-returns a `PrecompileUpgrade` struct. This function is used in the Ethereum API and can give a user
-information about what precompiles are enabled at a certain block timestamp. For more information
-about our Ethereum API, check out this
-[link](https://docs.avax.network/apis/avalanchego/apis/subnet-evm). The below code snippet can be
-copied and pasted to overwrite the default code.
+	_ "github.com/ava-labs/subnet-evm/precompile/contracts/txallowlist"
 
-<!-- markdownlint-disable MD013 -->
+	_ "github.com/ava-labs/subnet-evm/precompile/contracts/feemanager"
 
-```go
-func (c *ChainConfig) GetActivePrecompiles(blockTimestamp *big.Int) PrecompileUpgrade {
-	pu := PrecompileUpgrade{}
-	if config := c.GetContractDeployerAllowListConfig(blockTimestamp); config != nil && !config.Disable {
-		pu.ContractDeployerAllowListConfig = config
-	}
-	if config := c.GetContractNativeMinterConfig(blockTimestamp); config != nil && !config.Disable {
-		pu.ContractNativeMinterConfig = config
-	}
-	if config := c.GetTxAllowListConfig(blockTimestamp); config != nil && !config.Disable {
-		pu.TxAllowListConfig = config
-	}
-	if config := c.GetFeeConfigManagerConfig(blockTimestamp); config != nil && !config.Disable {
-		pu.FeeManagerConfig = config
-	}
-  	if config := c.GetHelloWorldConfig(blockTimestamp); config != nil && !config.Disable {
-		pu.HelloWorldConfig = config
-	}
-	// ADD YOUR PRECOMPILE HERE
-	// if config := c.{YourPrecompile}Config(blockTimestamp); config != nil && !config.Disable {
-	// 	pu.{YourPrecompile}Config = config
-	// }
+	_ "github.com/ava-labs/subnet-evm/precompile/contracts/rewardmanager"
 
-	return pu
-}
+	_ "github.com/ava-labs/subnet-evm/precompile/contracts/helloworld"
+	// ADD YOUR PRECOMPILE HERE
+	// _ "github.com/ava-labs/subnet-evm/precompile/contracts/yourprecompile"
+)
 ```
 
-<!-- markdownlint-enable MD013 -->
-
-Done! All we had to do was follow the comments.
+### Step 6: Add Config Tests
 
-### Step 5: Add Precompile Upgrade
+Let's add some unit tests to make sure our precompile is configured correctly. Config tests will
+be under `/precompile/contracts/helloworld/config_test.go`. There are mainly 2 functions we need
+to test: `Verify` and `Equal`. `Verify` checks if the precompile is configured correctly. `Equal`
+checks if the precompile is equal to another precompile. You can check other `config_test.go` files
+in the `/precompile/contracts` directory for examples. For the `HelloWorld` precompile, you can
+check the code in [here](https://github.com/ava-labs/subnet-evm/blob/helloworld-official-tutorial-v2/precompile/contracts/helloworld/config_test.go)
```

Comment:
```suggestion
Let's add some unit tests to ensure our precompile is configured correctly. Config tests will
be under `/precompile/contracts/helloworld/config_test.go`. We need to test mainly two functions: `Verify` and `Equal.` `Verify` checks if the precompile is configured correctly. `Equal`
checks if the precompile is equal to another precompile. For example, you can check other `config_test.go` files
in the `/precompile/contracts` directory. For the `HelloWorld` precompile, you can
check the code [here](https://github.com/ava-labs/subnet-evm/blob/helloworld-official-tutorial-v2/precompile/contracts/helloworld/config_test.go)
```

Reply:
Is this a whitespace diff suggestion? I intentionally split those lines to comfort the enforced MD rules.

Reply:
sorry for the big suggestion but it was mainly for 2 - >two

Reply:
ah thanks. changed it.